paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormViewModel.kt

```diff
@@ -602,27 +577,10 @@ internal class USBankAccountFormViewModel @Inject internal constructor(
         }
     }
 
-    private fun updatePaymentSelection(
-        resultIdentifier: ResultIdentifier,
-        bankName: String?,
-        last4: String?
-    ) {
-        if (bankName == null || last4 == null) return
-
-        val paymentSelection = createNewPaymentSelection(
-            resultIdentifier = resultIdentifier,
-            last4 = last4,
-            bankName = bankName,
-        )
-
-        _result.tryEmit(paymentSelection)
-        shouldReset = true
-    }
-
     private fun createNewPaymentSelection(
         resultIdentifier: ResultIdentifier,
-        last4: String,
-        bankName: String,
+        last4: String?,
+        bankName: String?,
```

Comment:
We previously expected these values, even though they can legitimately be `null`.